daml-assistant/daml-helper/BUILD.bazel

```diff
@@ -62,3 +62,31 @@ package_app(
     binary = ":daml-helper",
     visibility = ["//visibility:public"],
 )
+
+da_haskell_test(
```

Comment:
I know that we have some integration tests already for this but the integration tests are already very slow, badly cached and a nightmare to work with so I don’t want to extend them further.

Reply:
That sounds very reasonable to me.